http/action/src/test/java/io/knotx/databridge/http/action/HttpActionFactoryTest.java

```diff
@@ -44,6 +44,7 @@ void expectExceptionWhenDoActionProvided(Vertx vertx) throws Throwable {
   void shouldCreateHttpActionWhenProperlyConfigured(Vertx vertx) {
     HttpActionFactory actionFactory = new HttpActionFactory();
     JsonObject config = new JsonObject();
+    config.put("logLevel", "info");
```

Comment:
Is it mandatory? What is the default value?

Reply:
error log level is by default

Reply:
So why do you try to set "info" in `shouldCreateHttpActionWhenProperlyConfigured`?